source/elements/oneDPL/source/parallel_api/execution_policies.rst

```diff
@@ -180,5 +178,53 @@ as the template argument, otherwise unspecified.
 Return a policy object constructed from ``policy``, with a new kernel name provided as the template
 argument. If no policy object is provided, the new policy is constructed from ``dpcpp_default``.
 
+Execution Policy Type Trait
++++++++++++++++++++++++++++
+
+oneDPL provides type trait utilities to detect at compile time its execution policy types for the purpose of
+function overload resolution:
+
+.. code:: cpp
+
+  // Defined in <oneapi/dpl/execution>
+
+  namespace oneapi {
+    namespace dpl {
```

Comment:
Considering we are encouraging people to use the version at `oneapi::dpl::`, should we list that one before going one level deeper into `oneapi::dpl::execution::`?

At a quick glance, a user may just use the first version which is listed, and not read the note below.

Reply:
Why not; reordered.